client/src/pages/ProfileCompleted.js

```diff
@@ -1,6 +1,7 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React, { useEffect } from "react";
+import { Link, useLocation, withRouter } from "react-router-dom";
```

Comment:
Hey @kterry1  - I looked through everything else and tried this and I'm good. 

But I forgot to mention the original uses `withRouter` - but we should use `useLocation` or `withRouter` not both. There is an indication `withRouter` might be retired - https://reacttraining.com/blog/react-router-v5-1/ - so let's use `useLocation`.


```suggestion
import { Link, useLocation } from "react-router-dom";
```